source/CLEO_SDK/III.CLEO.h

```diff
@@ -38,20 +38,24 @@ union tScriptVar
 	DWORD dVar;
 };
 
+#pragma pack(push, 1)
```

Comment:
what does it affect, if we don't do it?

Reply:
Hmm, yep, it changes nothing (at leas as long as the structure is only 1 byte in size).